### PR TITLE
Beautify deviceflow prompt

### PR DESCRIPTION
**Pull request by ByronHsu**
## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Complete description
To make deviceflow prompt more prominent

Note: for why `verification_url_completed` is removed, check https://github.com/flyteorg/flytekit/pull/1611

image

### CODE REVIEW

File: flytekit/clients/auth/authenticator.py [hunk 1]
```diff
@@ -224,12 +224,8 @@ def __init__(
 
     def refresh_credentials(self):
         resp = token_client.get_device_code(self._device_auth_endpoint, self._client_id, self._audience, self._scope)
-        print(
-            f"""
-To Authenticate navigate in a browser to the following URL: {resp.verification_uri} and enter code: {resp.user_code}
-OR copy paste the following URL: {resp.verification_uri_complete}
-        """
-        )
+        text = f"To Authenticate, navigate in a browser to the following URL: {click.style(resp.verification_uri, fg='blue', underline=True)} and enter code: {click.style(resp.user_code, fg='blue')}"
```
Comment (kumare3) [inline flytekit/clients/auth/authenticator.py:227]: Actually let's use rich console

**Review by kumare3 — COMMENTED** [review]
I think we should start using rich console everywhere. It adjusts automatically to env like jupyter

File: flytekit/clients/auth/authenticator.py (hunk 1 above)
Comment (ByronHsu) [reply flytekit/clients/auth/authenticator.py:227]: Do you mean using the [logger](https://github.com/flyteorg/flytekit/blob/e44b8027bd0f59c0f90eb5f86bfa181aebad0d74/flytekit/loggers.py#L18)? Looks kinda weird  
image

Comment (kumare3) [reply flytekit/clients/auth/authenticator.py]: ya not the logger, i meant rich.console. but let it be what it was, can you revert and i can approve!

Comment (ByronHsu) [reply flytekit/clients/auth/authenticator.py:227]: i just reverted

**Comment by kumare3** [comment]
Some build fails?

**Review by pingsutw — APPROVED** [review]
LGTM, could you add a screenshot in the description?

**Review by pingsutw — APPROVED** [review]

**Pull request merged**